### PR TITLE
🚀 Release/v0.3.3

### PR DESCRIPTION
**Pull request by jasonbahl**
# 🚀 Release Notes

## (Potential) Breaking Changes
If you were using `WPGraphQL::$allowed_post_types` or `WP_GraphQL::$allowed_taxonomies` in your custom code, we recommend updating to use `WPGraphQL::get_allowed_post_types()` and `WPGraphQL::get_allowed_taxonomies()` instead, as some changes in this PR could potentially cause you issues. 

## Bugfixes
This release fixes several regressions that snuck by in the v0.3.0 release cycle.

- #812 Addresses #811, where the `$where` args for querying menus was not properly being respected post v0.3.0. Thanks for reporting and your patience as we resolved, @josephCarrington!
- #814 Fixed a regression with Custom Post Type id fields not working properly. Thanks @benallfree!
- #816 addressed #815, where static definitions of `WPGraphQL::$allowed_post_types` were causing funkiness. 
- #824 addresses #823, where Custom Taxonomy terms were loading funky in some cases. Thanks for reporting and working through this, @paulisloud, @seagyn!
- Fix to `hideEmpty` description on Term connection args. Thanks @epeli!

## New!!
- Updates to Docker testing environment. Thanks @mngi-arogers!
- #831  New "success" field returned for `createComment` payload so the mutation can respond with _something_ indicating the `createComment` succeeded, even if the user doesn't have access to get an unapproved comment returned as a result of the mutation. Thanks for working through this with me @paulisloud!

### CODE REVIEW

File: src/Data/Loader/TermObjectLoader.php [hunk 1]
```diff
@@ -66,9 +79,9 @@ public function loadKeys( array $keys ) {
 			 * them from the cache to pass through the model layer, or return null if the
 			 * object isn't in the cache, meaning it didn't come back when queried.
 			 */
-			$term_object                = get_term_by( 'term_taxonomy_id', $key );
+			$term_object                = $terms_by_id[ $key ];
```
Comment (hughdevore) [inline src/Data/Loader/TermObjectLoader.php]: 👍

File: src/Type/Object/PostObject.php [hunk 2]
```diff
@@ -137,6 +137,9 @@ function get_post_object_fields( $post_type_object ) {
 				'non_null' => 'Int',
 			],
 			'description' => __( 'The id field matches the WP_Post->ID field.', 'wp-graphql' ),
+			'resolve'     => function ( Post $post, $args, $context, $info ) {
```
Comment (hughdevore) [inline src/Type/Object/PostObject.php]: 👍

File: src/Type/Object/TermObject.php [hunk 3]
```diff
@@ -23,6 +23,9 @@ function register_taxonomy_object_type( $taxonomy_object ) {
 			$single_name . 'Id' => [
 				'type'        => 'Int',
 				'description' => __( 'The id field matches the WP_Post->ID field.', 'wp-graphql' ),
+				'resolve' => function( Term $term, $args, $context, $node ) {
```
Comment (hughdevore) [inline src/Type/Object/TermObject.php]: 👍

**Comment by paulisloud** [comment]
Tested on my end as well, working great! Thanks @jasonbahl!

**Pull request merged**